spec/features/affiliations_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature "Dodawanie Afiliacji" do
+
+  context "po zalogowaniu" do
+    include_context "admin login"
+
+
+    context "Afiliacje" do
+      before do
+      	Person.create!(name: "Andrzej", surname: "Kapusta", email: "a.kapusa@gmail.com", sex:
+                   "mężczyzna", roles: ['redaktor', 'recenzent'])
+      end
+
+      scenario "Sprawdzenie dodania poprawnej afiliacji" do
+        visit '/people/'
+        click_on("Kapusta")
+        click_on("Dodaj afiliację")
+
+		within("#new_affiliation_composite") do
+	        fill_in "Państwo", with: "Polska"
```

Comment:
Tak faktycznie jest to wina autocomplete. Zamiast `Państwo` proszę wpisać `affiliation_composite_country_autocomplete`.